docker/manager.py

```diff
@@ -712,9 +712,10 @@ class attributes:
 
         - _push_context (:obj:`Dict`):
 
-            contains mapping for docker.Images object from docker-py with correct releases type
+            contains mapping for docker.Images object from docker-py with correct
+            releases type
 
-    """
+    """  # noqa: LN001
```

Comment:
Is this file currently used? It seems like it doesn't even evaluate without errors.

Reply:
we are using everything related under docker for creating releases for our docker image.

Reply:
Hm, do I have the wrong docker version or something? Flake8 [complains](https://github.com/bentoml/BentoML/runs/4115558301?check_suite_focus=true) that `docker.DockerClient` and `docker.errors` don't exist

Reply:
@sauyon it seems to work with the latest version of docker python library 5.0.3, but for version 4.x, `docker.DockerClient` and `docker.errors` don't exist

Reply:
Could be because pylint detect the docker at our top level modules instead of the actual docker library.

Reply:
@aarnphm yes I got rid of that error by removing the `__init__.py` file under the `docker` directory